src/Controls/src/Core/FeatureFlags.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+namespace Microsoft.Maui.Controls
+{
+	internal static class FeatureFlags
```

Comment:
Would it make sense to name the type `RuntimeFeature` similarly to `System.Runtime.CompilerServices.RuntimeFeature` as it is essentially a runtime feature?

Reply:
I would also add a summary comment that links to an example in dotnet/runtime:

* https://github.com/dotnet/runtime/blob/main/docs/workflow/trimming/feature-switches.md